charts/virtual-kubelet/values.yaml

```diff
@@ -6,12 +6,18 @@ image:
 ## `provider` should be one of aws, azure, azurebatch, etc...
 provider: 
 nodeName: "virtual-kubelet"
-nodeTaint: "azure.com/aci"
 nodeOsType: "Linux"
 monitoredNamespace: ""
 apiserverCert: 
 apiserverKey: 
 
+taint:
+  enabled: true
+  key: virtual-kubelet.github.io/provider
+  value: "{{ .Values.provider }}"
+  ## `effect` must be `NoSchedule`, `PreferNoSchedule` or `NoExecute`.
+  effect: NoSchedule
```

Comment:
Enabling the taint and configuring its effect should definitely be options, but I'm ambivalent about allowing configuration of key and value from the helm chart.  Thoughts?

Reply:
I think my preference would be to default them like this, but allow people to override, if needed. I really think that the override is probably a SUPER special case and most people will chose to roll with this, but it doesn't hurt to allow it to be changed.

Reply:
Also, the disable is cool for down the road!